docs/conf.py

```diff
@@ -106,7 +98,7 @@
     "css/nvidia_footer.css",
 ]
 
-html_theme_options = {"display_version": True, "collapse_navigation": False, "logo_only": False}
```

Comment:
`display_version` was deprecated in the RTD theme 3.0.0 release: https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#confval-display_version

Note that the theme no longer includes the version in the sidebar, just a pulldown menu if multiple versions are available. It seems they didn't think about the case where we want to show the version. Hopefully they respond to this comment: https://github.com/readthedocs/sphinx_rtd_theme/pull/1601#issuecomment-2408718865